README.md

```diff
@@ -69,12 +69,24 @@ For use cases where you want to fetch the certificate data from a secure store,
 java_ks { 'broker.example.com:/etc/activemq/broker.ks':
   ensure              => latest,
   certificate_content => $certificate_data_fetched_from_secure_store,
-  private_key_content => $private_key_data_fetched_from_secure_store
+  private_key_content => $private_key_data_fetched_from_secure_store,
   password            => 'albatros',
   password_fail_reset => true,
 }
 ```
 
+**NOTE:** The sensitive fields like `password`, `certificate_content` and `private_key_content` can be deferred using the [Deferred](https://www.puppet.com/docs/puppet/7/template_with_deferred_values.html) function on Puppet Master and enable to execute on agent. This will avoid the sensitive values to be part of Puppet Catalog.
```

Comment:
```suggestion
**NOTE:** Sensitive fields like `password`, `certificate_content` and `private_key_content` can be deferred using the [Deferred](https://www.puppet.com/docs/puppet/7/template_with_deferred_values.html) function. This will ensure sensitive values are not present in the Catalog.
```